src/fu-util.c

```diff
@@ -884,13 +884,18 @@ fu_util_device_test_step(FuUtilPrivate *priv,
 {
 	JsonArray *json_array;
 	const gchar *emulation_url = NULL;
+	gboolean only_enumerate;
+
+	/* sometimes the emulation is HUGE and we do not want to run it in CI */
+	only_enumerate =
+	    json_object_get_boolean_member_with_default(json_obj, "only-enumerate", FALSE);
```

Comment:
this feels really hacky.  So now how would you run the full emulation?  No argument for fwupdmgr, no env variable to change it?  You'd need to adjust the json manually.

Reply:
Yup, it is. I'm trying find the root issue now, but it's hard as the emulation is over 100MB in size -- I'm not even sure we want to download that from the LVFS for every CI run. Maybe flip it around and have `enumerate-in-ci` and gate on the `CI=true` env var?

Reply:
Nuking, as we fixed the root cause.